### ISSUE TITLE
Attention Blank, is it because my progressive training schedule?

### ISSUE THREAD

**Issue by Vi22388** [fatchord/WaveRNN#223]
I'm new at this, my GPU is not very good so I decreased batch size.
my attention is empty after 10k steps, which shouldn't be normal.
I'm using LJSpeech dataset.
This is the second time I preprocessed everything and trained. (i reinstalled the dataset)

![image](https://user-images.githubusercontent.com/76446010/105724185-7e265b80-5f1f-11eb-9224-1ce9da630d1b.png)

Loss is around 1.0 at 10k steps
Are my settings wrong here? Does this not work?

![image](https://user-images.githubusercontent.com/76446010/105724483-d2c9d680-5f1f-11eb-9042-233bc787cbd9.png)

Or is my GPU just not good enough if I need to setup like this?
Thanks!

NOTE: I LOOKED AT THIS ISSUE ALREADY -> https://github.com/fatchord/WaveRNN/issues/154

**Comment by fatchord** [comment]
Hi, sometimes the alignment will fail randomly. I've never tried with batch size of 8 so that could be it. Maybe try finetuning on one of the pretrained models.

**Comment by Ahmad21A** [comment]
did you ever solve this?

**Comment by Vi22388** [comment]
> did you ever solve this?

Sorry for the lack of feedback. No I did not. when fatchord commented that it fails sometimes I tried it again 2 more times, but it 
just didn't work. Maybe my gpu is just not good enough, like I said, but I just moved on when I couldn't get results.
Any help would be appreciated anyways!

**Comment by Ahmad21A** [comment]
I think i found a solution, 
1) increase "r" from 7 to 12 in the tts_schedule in hparams.py.
2) go to models file>tacotron.py, and change line 200 from "scores = torch.sigmoid(u) / torch.sigmoid(u).sum(dim=1, keepdim=True)" to "scores = F.softmax(u, dim=1)".

got this from https://github.com/fatchord/WaveRNN/issues/154#issuecomment-567851857
